cc-pk-service/README.md

```diff
@@ -13,3 +13,8 @@
 Additional scripts: 
 1. Run ```npm run check``` to check the file for TypeScript errors/warnings with eslint
 2. Run ```npm run dev``` to run nodemon
+
+### Docker Guide 
```

Comment:
Maybe add note that you should add .env file before

Reply:
done